boards/px4/fmu-v4/nuttx-config/include/board.h

```diff
@@ -269,6 +269,10 @@
 #define GPIO_SPI1_MOSI  (GPIO_SPI1_MOSI_1|GPIO_SPEED_50MHz)
 #define GPIO_SPI1_SCK   (GPIO_SPI1_SCK_1|GPIO_SPEED_50MHz)
 
+#define DMACHAN_SPI1_RX DMAMAP_SPI1_RX_1
+#define DMACHAN_SPI1_TX DMAMAP_SPI1_TX_2
```

Comment:
This one conflicts with DShot (TIM1_UP). We can use `DMAMAP_SPI1_TX_1` and move the SDIO to stream 6 (`DMAMAP_SDIO_2`).

Reply:
Thanks, I haven't really looked at where things settled after dshot came in. We need to do a pass to get this in place for most of the boards. https://github.com/PX4/Firmware/pull/12436

Reply:
I've started capturing the full mapping in board_dma_map.h (next to board.h).

Reply:
The current state is here: https://docs.google.com/spreadsheets/d/1mFul5rEZUhfWSB8qSNtGBibBrFA8MPpm0KkxnSyVReI/edit#gid=0. Moving this in-tree is fine with me, it should just be easy to see what are the alternative options.